src/libraries/System.Text.Json/tests/Common/ConstructorTests/ConstructorTests.Cache.cs

```diff
@@ -11,9 +11,6 @@ public abstract partial class ConstructorTests
     {
         [Fact]
         [OuterLoop]
-#if BUILDING_SOURCE_GENERATOR_TESTS
-        [ActiveIssue("Needs JsonExtensionData support.")]
-#endif
         public async Task MultipleThreadsLooping()
```

Comment:
Nit: what is this test meant to be testing? If it's ensuring that the serialization infrastructure can cope with many serialization in parallel, that is generally handled by xunit's capacity to run multiple tests in parallel. If it's testing something else, it should be refelected in the test name.